La formazione per autodidatti

```diff
@@ -0,0 +1,77 @@
+Ora, zoomiamo sul nostro specifico interesse, l’auto-formazione: è ovvio che la discussione filosofica sin qui fatta si applica a tutti, autodidatti e/o persone che hanno seguito un corso di studi classico (università) ma per entrambi, la storia della creatività continua a valere.
+Di contro, è anche vero che seguire un percorso universitario non ha mai fatto male a nessuno ed offre il vantaggio di frequentare un ambiente nel quale è possibile confrontarsi con altre persone con la nostra stessa passione ed interessi, mentre se siamo autodidatti, queste possibilità sono un po’ più ridotte, ma non insuperabili.
+Resta il fatto, che in generale sarete “soli” davanti a libri, monitor, codice e… caffè!
+Ad ogni modo, a volte è “meglio soli che male accompagnati” e se siete veramente motivati ed appassionati alla materia, anche lo studio e la sperimentazione in solitaria non vi peserà più di tanto.
+
+Da non sottovalutare, nel 2023, che la pervasività e disponibilità H24 7/7 di una connessione ad internet è di grandissimo aiuto ad un autodidatta, perché le risorse che può reperire in rete, a gratis o con poca spesa, sono praticamente sterminate e –soprattutto agli inizi-, costituiscono un validissimo aiuto alla comprensione di argomenti che tutto sono meno che romantici.
+Superate le domande motivazionali, prima di mettervi all’opera, dovete porvi parecchie altre domande, stavolta di natura squisitamente più tecnica/tecnologica.
+
+Il termine “programmare” è un ombrello enorme che accoglie sotto di se’ innumerevoli sfumature di codice e dunque, sapere con la miglior precisione possibile a quale tipo di programmazione volete dedicarvi,  vi tornerà utilissimo per sfrondare l’inutile e farvi invece una lista utilissima degli strumenti dei quali dovreste impadronirvi.
+Non esiste una sola forma di programmazione ma qualunque applicativo (a meno che non vogliate scrivere solo script  bash di amministrazione), di solito ha una “interfaccia utente”, in genere ma non necessariamente anche grafica che è detta “**Frontend**” e tutto quello che succede quando un utente clicca su un pulsante del Frontend; questa parte, è nota come "**Backend**" o retrobottega.
+Il Backend non è visibile all’utente e quindi non viene applaudito e lodato come i Frontend ma state certi che se avrete fatto un buon lavoro –chessò?-, su un sistema bancario, l’utente finale vi sarà grato per avergli ad esempio evidenziato in maniera chiara e precisa problemi o anche opportunità legate al suo conto corrente.
+
+Fatta questa prima scelta, dovete preoccuparvi di capire su quali tecnologie volete formarvi e prepararvi.
+Un mio consiglio, opinione del tutto personale, è: evitate come la peste di gettarvi a corpo morto sui numerosi  “frame work” in circolazione.
+Che siano frame work frontend o back end, è essenziale che prima di utilizzarli vi impadroniate di ciò che c’è alla loro base.
```

Comment:
```suggestion
Che siano frame work frontend o backend, è essenziale che prima di utilizzarli vi impadroniate di ciò che c’è alla loro base.
```

Qui consigli di tenere una linea comune, quindi o parliamo di `front end` e `back end`, oppure parliamo di `frontend` e `backend`; in breve, eviterei di mixare le versioni